README.md

```diff
@@ -60,8 +175,9 @@ wfProxyClientBuilder.FlushIntervalSeconds(2);
 IWavefrontSender wavefrontSender = wfProxyClientBuilder.Build();
 ```
 
-### Create a WavefrontDirectIngestionClient
-To create a `WavefrontDirectIngestionClient`, you must have access to a Wavefront instance with direct data ingestion permission:
+### Option 2: Sending Data via Direct Ingestion
+To create a `WavefrontDirectIngestionClient`, you must have access to a Wavefront instance with [direct data ingestion permission](https://docs.wavefront.com/permissions_overview.html):
```

Comment:
Mmm. Maybe we need to mention Proxies permission above? 
And, I would capitalize Direct Data Ingestion permission (that's the name in the UI)

Reply:
Thank you! Good point!